tests/gh-api-helpers.js

```diff
@@ -0,0 +1,210 @@
+const { Octokit } = require('octokit')
+const axios = require('axios')
+const AdmZip = require('adm-zip')
+
+const ref = process.env.GITHUB_HEAD_REF || process.env.GITHUB_REF
+const owner = 'Kong'
+const repo = 'gateway-test-scheduler'
```

Comment:
Can these be taken from the environment instead?  Otherwise, it will be difficult to run the tests in forks.

Reply:
yes they can. Done.